docker/nginx/nginx.conf

```diff
@@ -62,5 +63,8 @@ http {
 
     #gzip  on;
 
+    # include skynet-portal-api header on every request
+    header_filter_by_lua 'ngx.header["Skynet-Portal-Api"] = os.getenv("SKYNET_PORTAL_API")';
```

Comment:
what's the reason you're doing this with `header_filter_by_lua` - wouldn't you able to accomplish the same thing simply by doing `add_header  Skynet-Portal-Api  {SKYNET_PORTAL_API};`

Reply:
Ok so I guess Nginx can't handle env variables like that. Lua it is then. In general though we should refrain from using `lua` if we don't absolutely have to.